examples/python/FullScriptSupport_Pandas/README.md

```diff
@@ -0,0 +1,45 @@
+
+If the parameter is of type _Dual_ the plugin will create two additional columns in the `q` data frame, with the string and numerical representation. The column names will have the base as the parameter name but will end with '_str' and '_num' respectively. For example, a parameter called `Bar` with datatype _Dual_ will result in three columns in `q`: `Bar`, `Bar_str` and `Bar_num`. `Bar` will contain strings and numerics, `Bar_str` will contain only strings and `Bar_num` only numerics.
+
+### TableDescription
+In the load script, when using the `Load ... Extension ...` syntax you can create the `TableDescription` message within the script. This can be useful if you for instance want to name, set tags, or change datatype of the fields you are sending back to Qlik. Read more about what metadata can be included in the `TableDescription` in in the [SSE_Protocol.md](../../../docs/SSE_Protocol.md).
+
+An instance of the `TableDescription` message is available from the script by the name `table`. To that instance you can add metadata according to the protocol. A few simple examples:
+
+- `table.name = "Table1"` sets the table name to be _Table1_
+- `table.fields.add(name="firstField", dataType=1, tags=["tag1", "tag2"])` adds a _numeric_ field called _firstField_ with the tags _tag1_ and _tag2_.
+
+Note that if a `TableDescription` is sent, the number of fields in the message must match the number of fields of data sent back to Qlik.
+
+### Result
+With the change to using the `exec` method to evaluate the script, there are some changes regarding what's possible to write in the script. See the Python documentation of `exec` [here](https://docs.python.org/3/library/functions.html#exec). One of them being that the `exec` method does not return anything. We must therefore set the result to a specific variable which we have chosen to call `qResult`. If nothing was set to the variable no data will be returned to Qlik. Note that `qResult` is not required to be a pandas data frame.  
+
+For example, if you want to return the same parameters as received from Qlik you can use the script `'qResult = q.values'`. Note that if I wrote `'qResult = q'` the entire data frame, including the column names as the first row, will be passed along to where the duals and BundledRows are created. This could result in an error if the column names are strings and you are supposed to return numerics.
+
+
+## Qlik documents
+An example document is given for Qlik Sense (SSE_Full_Script_Support_pandas.qvf) and it's the same as the original Full Script Support example but with modified scripts to work with the pandas implementation and the use of `exec`.
```

Comment:
We provide an example Qlik Sense document (SSE_Full_Script_Support_pandas.qvf). It's the same as the original Full Script Support example, but with modified scripts to work with the Pandas implementation and the use of `exec`.